templates/scale_up_resources.j2

```diff
@@ -52,11 +54,11 @@ ms msl_SAPHana_{{ sid }}_HDB{{ instance }} rsc_SAPHana_{{ sid }}_HDB{{ instance
 
 {% if data.platform == "azure" %}
 
-primitive rsc_nc_{{ sid }}_HDB{{ instance }} anything \
+primitive rsc_socat_{{ sid }}_HDB{{ instance }} anything \
```

Comment:
Can we rename this resource as such? Just thinking where else the resource name is used

Reply:
I have changed the name in all the dependencies.
We should do the same in the sapnwbootstrap-formula